src/MongoDB.Driver.Core/Core/Misc/DnsClientWrapper.cs

```diff
@@ -38,6 +38,23 @@ public DnsClientWrapper()
         public LookupClient LookupClient => _lookupClient;
 
         // public methods
+        public string GetHostNameWithReverseDnsLookup(string hostNameOrAddress)
```

Comment:
Should this method be called `GetCanonicalizedHostName`?

Or `GetCanonicalHostName`? (don't think the "ized" suffix is needed).

Reply:
`GetCanonicalHostName` - no, because the required value is not simple `CanonicalName`. I described it a little bit here: https://github.com/DmitryLukyanov/mongo-csharp-driver/pull/84#discussion_r351274338
Let me know if you still any questions about it.

Reply:
`GetCanonicalizedHostName` - sounds good to me. Will do it, if no objections